rosario_angelica/countries.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Countries of the World</title>
+  <link rel="stylesheet" href="design.css">
+</head>
+<body>
+  <div class="section">
+    <h1>Countries of the World</h1>
+    <div class="search-container">
+      <input type="text" id="country-input" placeholder="Enter country name">
```

Comment:
Hi, good morning! Would you mind making new changes in naming ids? All IDs should be in snake_case, for example:
❌ id="country-input"
✅ id="country_input"

see attached link: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR15hTWkgykAvEQujS74FHgJUa9UxiKSLF6C5MzuxYQZct0Vu7jQYWUyQAI_aem_AfbRDN52Fnw7TRXXR4uiXndLIWpv01TwxZMo7TvdPUV2YlzDnVgyePFxmQR_sexUJd0GnHMcLQM2qJdkeo00i445#slide=id.g24da8de92d7_0_5